cloud/azure/outputs.tf

```diff
@@ -1,3 +1,7 @@
 output "app_service_default_hostname" {
   value = "https://${azurerm_app_service.civiform_app.default_site_hostname}"
 }
+
+output "pg_server_privatelink_fqdn" {
+  value = azurerm_private_endpoint.endpoint.private_dns_zone_configs[0].record_sets[0].fqdn
```

Comment:
question (blocking): does terraform support storing `azurerm_private_endpoint.endpoint.private_dns_zone_configs[0].record_sets[0].fqdn` in a variable so the variable can be referenced here and on L75 of main.tf?

Reply:
It does. That's what [local values](https://www.terraform.io/docs/language/values/locals.html) are used for. Added this to `locals.tf`